config/install/views.view.apigee_api_catalog.yml

```diff
@@ -3,19 +3,18 @@ status: true
 dependencies:
   config:
     - core.entity_view_mode.apidoc.card
-    - field.storage.apidoc.field_categories
```

Comment:
I'm not sure if this should be removed.... Going to try to find out.

Reply:
I asked today, and it seems this field should remain.

Reply:
@arshad and I figured out this was left over from older `apigee_edge_apidocs` code, so this was a false alarm.  It's all good.